src/components/pages/AddFarmer/AddFarmer.jsx

```diff
@@ -0,0 +1,251 @@
+                    <option value="">NHIS</option>
+                    <option value="">National ID</option>
+                    <option value="">Others</option>
+                  </select>
+                </label>
+              </div>
+              <label for="">
+                ID Number: <input type="text" name="id_number" />{' '}
+              </label>
+              <label for="">
+                District: <input type="text" name="district" />{' '}
+              </label>
+              <label for="">
+                Region: <input type="text" name="region" />{' '}
+              </label>
+              <label for="">
+                Community: <input type="text" name="community_name" />{' '}
+              </label>
+              <label for="">
+                House Name: <input type="text" name=" house_name" />{' '}
+              </label>
+              <label for="">
+                House No: <input type="text" name=" house_number" />{' '}
+              </label>
+              <label for="">
+                Nearest Landmark: <input type="text" name="nearest_landmark" />{' '}
+              </label>
+              <label for="">
+                Phone 1: <input type="text" name="Phone_1" />{' '}
+              </label>
+              <label for="">
+                Phone 2: <input type="text" name="Phone_2" />{' '}
+              </label>
+              <label for="">
+                Education: <input type="text" />{' '}
+              </label>
+              <div>
+                <label for="">
+                  Educational Level:
+                  <select id="" name="education_level">
+                    <option value="">Tertiary</option>
+                    <option value="">SHS</option>
+                    <option value="">JHS</option>
+                    <option value="">Primary</option>
+                    <option value="">Not Educated</option>
+                  </select>
+                </label>
+              </div>
+              <label for="">
+                Occupation: <input type="text" name="occupation" />{' '}
+              </label>
+              <div>
+                <label for="">
+                  Expected Income Per Month:
+                  <select id="" name="expected_income_per_month">
+                    <option value="">Less than GHC 500</option>
+                    <option value="">501 to GHC 1,000</option>
+                    <option value="">More than GHC 1,000</option>
+                  </select>
+                </label>
+              </div>
+              <label for="">
+                Major Income Source: <input type="text" name="major_source_of_income" />{' '}
+              </label>
+              <label for="">
+                Amount: <input type="text" name="major_source_of_income" />{' '}
+              </label>
+              <label for="">
+                Major Income Source: <input type="text" name="minor_source_of_income" />{' '}
```

Comment:
I guess this should be "minor source of income" both times

Reply:
its just a draft pr